Dockerfile

```diff
@@ -41,4 +41,5 @@ ENV PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH \
 RUN HOMEBREW_NO_ANALYTICS=1 HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/core \
 	&& chown -R linuxbrew: /home/linuxbrew/.linuxbrew \
 	&& chmod -R o-w /home/linuxbrew/.linuxbrew \
+	&& chmod -R g+w /home/linuxbrew/.linuxbrew \
```

Comment:
Replace these two lines with `chmod -R g+w,o-w`. Is `brew doctor` happy with these permissions?

Reply:
No complaints from `doctor` about this.